### ISSUE TITLE
[Windows] Testpilot version - can't close installer window

### ISSUE THREAD

**Issue by tflidd** [owncloud/client#3642]
I downloaded this installer (there was no installer linked, so I "guessed" the link):
https://download.owncloud.com/desktop/testing/testpilotcloud-2.0.0.5408rc1-setup.exe

I went through the installer until the end, when I have the choice to launch the testpilot. I selected it to be opened but on "finish" the Window of the installer didn't close. So I unchecked the checkbox and clicked on "finish". There was the next screen, see screenshot:
![screenshot](https://cloud.githubusercontent.com/assets/8111200/9341111/32daadf6-45f4-11e5-837a-587683676b12.png)
In the background the client was already started.

OS: Win 7 (64 bit)

During start of the client, the antivirus (Avira Free Antivirus 15.0.12.408) was scanning something. I disabled the virus scanner temporarily but I still can't close the windows from the screen shot. I don't think it's related but you never know.

**Comment by danimo** [comment]
I cannot reproduce this issue on Win 7 or 8.1.

**Comment by tflidd** [comment]
I tried a different computer and it went through without a problem (Win 7 Home Premium 64 bit). I think the other system with problems was Win 7 Pro.

**Comment by danimo** [comment]
Closing as can't reproduce for now.

**Comment by tflidd** [comment]
I just installed the RC2-version on the computer it was working before (Win 7 Home Premium 64 bit), and I got exactly the same error. When finishing the installation (check-box starting client is active) and clicking nervously on finish, I can get to the window in the initial post.
The opening of the client is delayed by the virus scanner (Avira Antivirus Pro), probably the same on the initial system (Win 7 Pro 64 bit) with free Avira version. It seems like the delay causes the problem.

**Comment by tflidd** [comment]
When trying to kill the installer, I just saw that there are two sessions open:
![screenshot](https://cloud.githubusercontent.com/assets/8111200/9440515/ff5625d4-4a6f-11e5-8170-f8a744afa43c.png)

**Comment by tflidd** [comment]
I tried to reproduce it. There are always two processes testpilotcloud-2... during the installation process. But there was no error again.

I tried a little bit further. I used the nightly-version of the client (20150823), there were no problem. Then I started the installation (without previous deinstallation) of the latest nightly (20150824). The installer suggests to stop the running testpilot and to deinstall them. I accepted both, and the client was finally installed. And I got the same error.
![screenshot2](https://cloud.githubusercontent.com/assets/8111200/9440987/a19aea0c-4a73-11e5-920a-dcf146f72d3d.png)
This time I got the screenshot with the avira messages. They seem to upload the files for further checks. Redoing the same installation procedure does not cause this error.
I suspect:
- if the file is new for avira, the file is uploaded to their service and causes the problem
- further installations run without problems because the file (checksum) is already known by their programs.
